modules/cluster-logging-release-notes-5.1.0.adoc

```diff
@@ -0,0 +1,68 @@
+`state=error: certificate verify failed (unable to get local issuer certificate)`
++
+Logs could not be forwarded to a Kafka broker with a certificate signed by an intermediate CA. This happened because the fluentd Kafka plugin could only handle a single CA certificate supplied in the `ca-bundle.crt` entry of the corresponding secret. The current release fixes this issue by enabling the fluentd Kafka plugin to handle multiple CA certificates supplied in the `ca-bundle.crt` entry of the corresponding secret. Now, logs can be forwarded to a Kafka broker with a certificate signed by an intermediate CA.
+(link:https://issues.redhat.com/browse/LOG-1218[*LOG-1218*], link:https://issues.redhat.com/browse/LOG-1216[*LOG-1216*])
+
+* Previously, while under load, Elasticsearch responded to some requests with an HTTP 500 error, even though there was nothing wrong with the cluster. Retrying the request was successful. This release fixes the issue by updating the index management cron jobs to be more resilient when they encounter temporary HTTP 500 errors. The updated index management cron jobs will first retry a request multiple times before failing.
+(link:https://issues.redhat.com/browse/LOG-1215[*LOG-1215*])
+
+* Previously, if you did not set the `.proxy` value in the cluster installation configuration, and then configured a global proxy on the installed cluster, a bug prevented Fluentd from forwarding logs to Elasticsearch. To work around this issue, in the proxy or cluster configuration, set the `no_proxy` value to `.svc.cluster.local` so it skips internal traffic. The current release fixes the proxy configuration issue. Now, if you configure the global proxy after installing an OpenShift cluster, Fluentd forwards logs to Elasticsearch.
+(link:https://issues.redhat.com/browse/LOG-1187[*LOG-1187*], link:https://bugzilla.redhat.com/show_bug.cgi?id=1915448[*BZ#1915448*])
+
+* Previously, the logging collector was creating more socket connections than necessary. With this fix, the logging collector re-uses the existing socket connection to send logs.
+(link:https://issues.redhat.com/browse/LOG-1186[*LOG-1186*])
+
+* Previously, as a cluster administrator, if tried to add or remove storage from an Elasticsearch cluster, the OpenShift Elasticsearch Operator (EO) incorrectly tried to upgrade the Elasticsearch cluster, displaying `scheduledUpgrade: "True"`, `shardAllocationEnabled: primaries`, and the change volumes. The current release fixes this issue, so the EO does not try to upgrade the Elasticsearch cluster.
++
+The EO status displays the following new status information to indicate when you have tried to make an unsupported change to the Elasticsearch storage that it has ignored:
++
+ - `StorageStructureChangeIgnored` when you try to change between using ephemeral and persistent storage structures.
+ - `StorageClassNameChangeIgnored` when you try to change the storage class name.
+ - `StorageSizeChangeIgnored` when you try to change the storage size.
+
```

Comment:
Missing a `+` here to keep the above note with this bullet for the known issue.

Reply:
Done